Packs/RecordedFutureV3/Integrations/RecordedFutureAlerts/RecordedFutureAlerts.py

```diff
@@ -0,0 +1,444 @@
+    def __init__(self, rf_client: Client):
+        self.client = rf_client
+
+    def test_module(self) -> None:
+        # This is the call made when pressing the integration Test button.
+        # Returning 'ok' indicates that the integration works like it suppose to and
+        # connection to the service is successful.
+        # Returning 'ok' will make the test result be green.
+        # Any other response will make the test result be red.
+
+        demisto_params = demisto.params()
+
+        # Validate first_fetch
+        first_fetch_str = str(demisto_params.get("first_fetch", ""))
+
+        if first_fetch_str.isnumeric():
+            first_fetch = int(first_fetch_str)
+        else:
+            raise ValueError("'first_fetch' parameter must be a number")
+        ninety_days_in_minutes = 90 * 24 * 60
+        if first_fetch > ninety_days_in_minutes:
+            raise ValueError("'first_fetch' parameter cannot be bigger than 90 days")
+
+        # Validate max_fetch
+        max_fetch_str = str(demisto_params.get("max_fetch", ""))
+        if max_fetch_str.isnumeric():
+            max_fetch = int(max_fetch_str)
+        else:
+            raise ValueError("'max_fetch' parameter must be a number")
+        if max_fetch > 50:
+            raise ValueError("'max_fetch' parameter cannot be bigger than 50")
+
+        try:
+            self.client.whoami()
+            return_results("ok")
+        except Exception as err:
+            message = str(err)
+            try:
+                error = json.loads(str(err).split("\n")[1])
+                if "fail" in error.get("result", {}).get("status", ""):
+                    message = error.get("result", {})["message"]
+            except Exception:
+                message = f"Unknown error. Please verify that the API URL and Token are correctly configured. RAW Error: {err}"
+            raise DemistoException(f"Failed due to - {message}")
+
+    def fetch_incidents(self) -> None:
+        response = self.client.fetch_incidents()
+        if isinstance(response, CommandResults):
+            # 404.
+            return_error("404 in fetch incidents")
+        alerts = response.get("alerts", [])
+        next_query_classic = response.get("next_query_classic", {})
+        next_query_playbook = response.get("next_query_playbook", {})
+        next_query = {
+            "next_query_classic": next_query_classic,
+            "next_query_playbook": next_query_playbook,
+        }
+
+        incidents = [
+            {
+                "name": alert.get("title"),
+                "occurred": alert.get("created"),
+                "dbotMirrorId": alert.get("id"),
+                "rawJSON": json.dumps(alert),
+            }
+            for alert in alerts
+        ]
+
+        demisto.incidents(incidents)
+        demisto.setLastRun(next_query)
+        return None
+
+    def alert_search_command(self) -> Union[dict, list[CommandResults]]:
+        return self.client.alert_search()
```

Comment:
in case nothing has return, return a sentence stating that, do not return an error